app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -26,7 +26,7 @@
         app:cardElevation="4dp"
         app:rippleColor="@color/gt_blue"
         app:strokeColor="@{category == selectedCategory ? @color/gt_blue : @color/white}"
-        app:strokeWidth="@{category == selectedCategory ? 10f : 0f}">
+        app:strokeWidth="@{category == selectedCategory ? 2f : 0f}">
```

Comment:
this should be `2dp` and not `2px`, you will need to use a dimension value to actually resolve it within a binding expression.